examples/app-secaggplus/server.py

```diff
@@ -0,0 +1,46 @@
+import flwr as fl
+from flwr.common import Context
+from flwr.server import Driver, LegacyContext
+from flwr.server.workflow import SecAggPlusWorkflow
+from workflow_with_log import SecAggPlusWorkflowWithLogs
+
+
+# Define strategy
+strategy = fl.server.strategy.FedAvg(
+    fraction_fit=1.0,  # Select all available clients
+    fraction_evaluate=0.0,  # Disable evaluation
+    min_available_clients=3,
+    fit_metrics_aggregation_fn=lambda _: {},
+)
+
+
+# Run via `flower-server-app server_workflow:app`
+app = fl.server.ServerApp()
+
+
+@app.main()
+def main(driver: Driver, context: Context) -> None:
+    # Construct the LegacyContext
+    context = LegacyContext(
+        state=context.state,
+        config=fl.server.ServerConfig(num_rounds=3),
+        strategy=strategy,
+    )
+
+    # Create the workflow
+    workflow = fl.server.workflow.DefaultWorkflow(
+        fit_workflow=SecAggPlusWorkflowWithLogs(
+            num_shares=3,
+            reconstruction_threshold=2,
+            timeout=5,
+        )
+        # # Use the following code in practice
+        # fit_workflow=SecAggPlusWorkflow(
+        #     num_shares=3,
+        #     reconstruction_threshold=2,
+        #     timeout=5,
+        # )
```

Comment:
Should we add logs to the built-in `SecAggPlusWorkflow` and set the log level to `DEBUG`? That way they won't show up for regular users when they don't change the log level

Reply:
I would not recommend doing so. The logs are tailored for this example only. In practice, it will print too much info that can occupy the whole console.

However, I think we can move some logs to to the built-in `SecAggPlusWorkflow`, e.g., the logs notifying the start of each stage.